libxml2.yaml

```diff
@@ -0,0 +1,75 @@
+package:
+  name: libxml2
+  version: 2.10.3
+  epoch: 0
+  description: XML parsing library, version 2
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: MIT
+
+secfixes:
+  2.10.3-r0:
+    - CVE-2022-40303
+    - CVE-2022-40304
```

Comment:
Is this correct?  If we add a new package with a secfix for the same version we are publishing, should we include these here?  
Also I've not included secfixes for previous versions?

Reply:
yes, that seems fine